src/LogBackend.cxx

```diff
@@ -9,11 +9,12 @@
 #include "Version.h"
 #include "config.h"
 
+#include <fmt/chrono.h>
```

Comment:
This commit not only adds configurable time formats, but also migrates to fmt. Please don't do two things in one commit! If you believe fmt is better, you can suggest doing that.